fastlane/metadata/android/cs/full_description.txt

```diff
@@ -0,0 +1,4 @@
+Využijte svůj telefon nebo tablet ke studiu nemocí, předpovídání globálního oteplování, nebo k objevení pulsarů! BOINC umí využít volný výpočetní výkon vašeho zařízení se systémem Android ke zpracování úkolu pro vědecko-výzkumné projekty. Můžete si vybrat z projektů v mnoha vědních oborech, od předních akademických institucí. BOINC počítá pouze tehdy, když je váš přístroj připojený na nabíječku a nabitý, nebude vám tak vybíjet baterii. Také přenáší data pouze přes WiFi, čímž šetří váš datový limit.
+
+Supports:
+Yoyo@Home, World Community Grid, PrimeGrid, Enigma@Home, OProject@Home, theSkyNet POGS, Asteroids@Home, and Einstein@Home
```

Comment:
theSkyNet POGS is not available anymore. 
I didn't find OProject@Home, I think it not available anymore.
PrimeGrid do not support android.
You can add WUProp@Home, Universe@Home, Rosetta@home, Moo! Wrapper, LHC@home

Reply:
Ашчув

Reply:
You miss some projects.
This is more complete list:

Asteroids@home, Einstein@home, LHC@home, Moo!Wrapper, Rosetta@home, Universe@home, World Community Grid, Yoyo@home, Enigma@Home, Asteroids@Home, WUProp@Home

Reply:
I copied from the list of Projects that are available on arm64 (that is the most popular architecture of Android mobile devices)

Reply:
Maybe, but this list is more complete. The other 3 projects also support android.

Reply:
Asteroids@home is the first in my list, Enigma@home is dead for 3 years or more, WUProp@home is not listed in the official list.

Reply:
Yes you are right.
 I am sorry about Asteroids@home is already in your list. 
 I didn't know that Enigma@home is dead, I just log in to their website and check apps.php.
 WUProp@home is not in official site, I don't know why. But all my android phone are running it. 
 Why we don't say we support it?

Reply:
Because WUProp@home just collects statistics about other Projects and does no science?

Reply:
I think is also important. Collect statistics that help people make decisions how to invest their science project (For example if it worth doing gpu project on mobile) it also science.

Reply:
Now in the information age, and many jobs like data science, collect data Is a real science professions.

Reply:
I do not make such decision and have no info about this projects and why it's not added to the official list. If it will be added - I'll happily update this description too